backend/src/services/v2/release.ts

```diff
@@ -55,6 +59,19 @@ async function validateRelease(user: UserDoc, model: ModelDoc, release: ReleaseD
         )
       }
     }
+
+    const uniqueDuplicates = [...new Set(findDuplicates(fileNames))]
+    if (uniqueDuplicates.length) {
+      throw BadReq(
+        `Releases cannot have multiple files with the same name.  The duplicates in this file are: '${JSON.stringify(
```

Comment:
Should models have multiple files with the same name? When a user creates a release the UI uploads their files first and then creates the release with those files. Throwing this error from the second request in this process might get a little messy. I assume upon receiving this error a user would normally rename the file and then attempt to again to create the release but under the hood that would duplicate all the files from the first attempt to create the release?